uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/partials/_forms.scss

```diff
@@ -1,3 +1,63 @@
+.modal-body {
+  .form-group {
+    opacity: 2;
+
+    .ui-select-container.no-border {
+      * {
+        border: none !important;
+        box-shadow: none !important;
+      }
+
+      .ui-select-choices {
+        border: 1px solid #333333 !important;
```

Comment:
It was already like that, and soon this bootstrap version will be consolidated with _bootstrapcivicrm_, but it's fixed now